201903_highres_foregrounds/README.md

```diff
@@ -0,0 +1,40 @@
+High resolution foregrounds with spectral index not varying spatially
+=====================================================================
+
+Release date March 2019
+
+The last version of this file is [available on Github](https://github.com/simonsobs/map_based_simulations/tree/master/201903_highres_foregrounds).
+The same folder contains all the configuration files used and the scripts to create SLURM jobs.
+
+## Input components
+
+Each component is saved separately, all maps are in `uK_CMB`, IQU, single precision (`float32`)
+
+Dust, synchrotron, free-free and anomalous microwave emission using the "0" models, i.e. `SO_d0`, `SO_s0`, `SO_f0`, and `SO_a0`, see more details in [in the documentation](https://so-pysm-models.readthedocs.io/en/0.2.dev/highres_templates.html#details-about-individual-models).
+
+## Available maps
+
+HEALPix maps at high resolution (nside 4096) and low resolution (nside 512), these models are deterministic, so we have
+a set for each resolution for all channels.
+
+**Location at NERSC**:
+
+    /project/projectdirs/sobs/v4_sims/mbs/201903_highres_foregrounds
```

Comment:
@NicolettaK I produced the NSIDE 512 simulations, can you check them?

They are at `/global/cscratch1/sd/zonca/simonsobs/mapsims_runs/201903_highres_foregrounds`